content/en/docs/ct-logs.html

```diff
@@ -11,7 +11,15 @@
 
 <p><a href="https://www.certificate-transparency.org/what-is-ct">Certificate Transparency (CT)</a> is a system for logging and monitoring the issuance of TLS certificates. CT greatly enhances everyone's ability to monitor and study certificate issuance, and these capabilities have led to numerous improvements to the CA ecosystem and Web security. As a result, CT is rapidly becoming critical infrastructure.</p>
 
-<p>Let's Encrypt submits all certificates we issue to CT logs. We also operate a CT log ourselves, which all publicly trusted certificate authorities are welcome to submit to.</p>
+<p>Let's Encrypt submits all certificates we issue to CT logs. We also operate several CT logs ourselves, which all publicly trusted certificate authorities are welcome to submit to.
+
+<ul>
+    <li><a href="https://letsencrypt.org/2019/05/15/introducing-oak-ct-log.html">Oak</a> is our production log and will eventually be incorporated in browser CT programs.</li>
+    <li>Testflume will never be incorporated in browser CT programs. It is used strictly to test new versions of <a href="http://github.com/google/trillian">Trillian</a> and <a href="https://github.com/google/certificate-transparency-go">certificate-transparency-go</a> software. The <a href="{{< relref "/docs/staging-environment.md" >}}">staging environment</a> submits all certificates to testflume. We welcome other CAs to send us their test root certificates and use Testflume.</li>
```

Comment:
Let's remove the "strictly" here, since we use it for three purposes:
 - To test new Trillian versions
 - To submit certificates from our staging ACME API
 - To allow other CAs to submit certificates from their test environment.

We should also link somewhere for "send us." Probably the contact page - it doesn't have anyplace clearly for the purpose, but until / unless we set up a specific alias, it's the best we've got.

We should also mention that Testflume accepts all the normal trusted roots as well, and that we submit certificates from our production ACME API to it so that it is always subjected to a "typical" amount of load and grows at a similar rate to Oak.

Perhaps with all this info, Testflume deserves a separate heading?